console/src/components/Services/Data/TableBrowseRows/ViewTable.js

```diff
@@ -201,6 +201,7 @@ class ViewTable extends Component {
         updateInvocationFunction={this.updateInvocationFunction.bind(this)}
         triggeredRow={triggeredRow}
         triggeredFunction={triggeredFunction}
+        location={this.props.location}
```

Comment:
Could you destruct this variable from props in here: https://github.com/hasura/graphql-engine/blob/9afe08ce094fbe9d2d009f826c008ce44d9c8c09/console/src/components/Services/Data/TableBrowseRows/ViewTable.js#L142
Just for the consistency.